lib/workers/repository/process/write.ts

```diff
@@ -108,6 +152,12 @@ export async function writeUpdates(
         ? branchFingerprint
         : branchState.branchFingerprint;
 
+    if (res?.commitSha) {
+      branchState = {
+        ...branchState,
```

Comment:
Which fields exist here which aren't in the cache?

Reply:
None, lapse in judgement.